api/src/main/java/org/openmrs/module/fhir2/api/impl/FhirEncounterServiceImpl.java

```diff
@@ -48,4 +50,14 @@ public Encounter getEncounterByUuid(String uuid) {
 		return dao.searchForEncounters(date, location, participant, subject).stream().map(translator::toFhirResource)
 		        .collect(Collectors.toList());
 	}
+	
+	@Override
+	protected FhirDao<org.openmrs.Encounter> getDao() {
+		return dao;
+	}
+	
+	@Override
+	protected OpenmrsFhirTranslator<org.openmrs.Encounter, Encounter> getTranslator() {
+		return translator;
+	}
```

Comment:
Rather than explicitly generating these fields, I would add the annotation `@Getter(AccessLevel.PROTECTED)` right above the annotation for `@Setter` at the top of the class.

Reply:
yes sure ,,,thanks

Reply:
have added it